### ISSUE TITLE
Can't send a message w/binary attribute and/or more than one attribute

### ISSUE THREAD

**Issue by brainoutsource** [adamw/elasticmq#53]
Hello!

I'm trying to send a message having two String attributes and a message having at least one Binary attribute.

For example, this request fails with response code 500:

```
{  
  "Action":[  
    "SendMessage"
  ],
  "Version":[  
    "2012-11-05"
  ],
  "MessageBody":[  
    "Test message"
  ],
  "MessageAttribute.1.Name":[  
    "stringAttribute"
  ],
  "MessageAttribute.1.Value.StringValue":[  
    "someString"
  ],
  "MessageAttribute.1.Value.DataType":[  
    "String"
  ],
  "MessageAttribute.2.Name":[  
    "binaryAttribute"
  ],
  "MessageAttribute.2.Value.BinaryValue":[  
    "c29tZVN0cmluZw=="
  ],
  "MessageAttribute.2.Value.DataType":[  
    "Binary"
  ]
}
```

However, a message having only one String attribute is sent fine.

Unfortunately, I have no logs from the ElasticMQ, the only thing's visible from the client side is `500 There was an internal server error.`

**Comment by adamw** [comment]
Currently only String attributes are supported - adding binary attributes support shouldn't be a problem, but it was never done. Maybe you'd like to try a PR? :)

**Comment by brainoutsource** [comment]
Working on it

**Comment by adamw** [comment]
Awesome! :)

**Comment by adamw** [comment]
Fixed in 0.8.9 as far as I know
